recipes_source/recipes/loading_data_recipe.py

```diff
@@ -1,32 +1,33 @@
 """
-Loading data in PyTorch
+PyTorch에서 데이터 불러오기
 =======================
-PyTorch features extensive neural network building blocks with a simple,
-intuitive, and stable API. PyTorch includes packages to prepare and load
-common datasets for your model.
+PyTorch는 인공신경망을 만드는데 필요한 다양한 기본 요소를 제공하고 있고, 간단하고, 직관적이고,
+안정적인 API로 쉽게 이용할 수 있습니다. PyTorch에는 공용 데이터셋을 쉽게 사용할
+수 있도록 도와주는 패키지를 포함하고 있습니다.
 
-Introduction
+소개
```

Comment:
```suggestion
개요
```
기존 번역에서 Introduction을 개요로 번역하고 있는데, 소개가 더 바람직할까요?

Reply:
개요 좋아요!